azurerm_logic_app_standard: add content_share_force_disabled property (#32212)
terraform-providers/terraform-provider-azurerm: website/docs/r/logic_app_standard.html.markdown

* `vnet_content_share_enabled` - (Optional) Specifies whether allow routing traffic between the Logic App and Storage Account content share through a virtual network. Defaults to `false`.

> * `content_share_force_disabled` - (Optional) Should the settings for linking the Logic App to storage (`WEBSITE_CONTENTAZUREFILECONNECTIONSTRING` and `WEBSITE_CONTENTSHARE`) be suppressed? Defaults to `false`. This is useful when deploying the Logic App to an App Service Environment (ASE) where the recommendation is to use the ASE internal storage account.

Reviewer: please use `content_share_force_enabled`

```suggestion
* `content_share_force_enabled` - (Optional) Specifies weather the settings for linking the Logic App to storage (`WEBSITE_CONTENTAZUREFILECONNECTIONSTRING` and `WEBSITE_CONTENTSHARE`) be suppressed? Defaults to `true`. 
```

Author: I think the property should be disabled to be the same as linux_function_app 
https://github.com/hashicorp/terraform-provider-azurerm/blob/12ee3f351b332528566b84fd557ecdd957ba9c3a/website/docs/d/linux_function_app.html.markdown?plain=1#L62

Reviewer: sorry did not link to the document, new properties shall follow the naming guide: https://github.com/hashicorp/terraform-provider-azurerm/blob/0393c68bad938e15f8a47a3b07b34204d4b07384/contributing/topics/reference-naming.md?plain=1#L42

Author: can I change it to "use_local_disk_on_the_worker_enabled" or "use_local_disk_enabled" 
"prevent_content_share_enabled"

to keep the default value false and reduce the change

Reviewer: IMO changing this to `content_share_force_enabled` is better, since it keeps the consistency with Portal and Azure documents.  the other names might be easier to understand but may confuses when a user saw the property on above places but can not find proper property in azureRM provider

Author: if content_share_force_enabled is used , it should default to true. Therefore, I will only apply the logic if it exists "and" is explicitly set to false. is that ok ?

Reviewer: sounds great
